### ISSUE TITLE
Saline

### ISSUE THREAD

**Issue by GGlut** [RMC-14/RMC-14#4791]
## Description
We can make upstream Saline

**Reproduction**
Just sink water and salt from the saltshakers in the kitchen

**Screenshots**

**Additional context**

**Comment by Vermidia** [comment]
Didn't whisper make this require like...4000 water and 1000 salt? How are you getting that in a single container?

**Comment by GGlut** [comment]
yes whipser changed it to require massive quantities. I have 0 idea why it just ignores that change.

**Comment by Vermidia** [comment]
It sure does...at this point we might just need to comment out the reaction or something tbh

**Comment by Boaz1111** [comment]
Well the recipes just define ratios for chems, not minimum quantities.

**Comment by Rainbeon** [comment]
Boaz is correct pretty sure, the recipes just defines the ratio, so that you can still mix chems at uneven quantities (for example, if you need 4 of x and 1 of y, but have only 3 of x, it should still react with all of the x, leaving 0.25 y leftover)

**Comment by GGlut** [comment]
yep it just needs "quantized: true" to have a minimum reaction amount like gunpowder

**Comment by Aurallianz** [comment]
#4810, honestly, let's just solve all of these bugs with a shotgun approach.

**Comment by Vermidia** [comment]
Fixed by #6011
